src/components/learner-credit-management/BudgetAssignmentsTable.jsx

```diff
@@ -112,16 +119,16 @@ const BudgetAssignmentsTable = ({
           disableFilters: true,
         },
       ]}
-      additionalColumns={[
+      additionalColumns={(!isRetired) ? [
         {
           Header: '',
           Cell: AssignmentRowActionTableCell,
           id: 'action',
         },
-      ]}
-      tableActions={[
+      ] : []}
+      tableActions={(!isRetired) ? [
         <AssignmentsTableRefreshAction refresh={fetchTableData} />,
-      ]}
+      ] : []}
```

Comment:
[nit/suggestion] similar to comment above, the logic should be resolved outside of the component. Since they are so similar you create a function that returns an object based on the value of `isRetired`

Pseudocode Ex.
```js
const budgetAssignmentsTableData = (isRetired) => {
if(!isRetired) {
return {
    tableActions: [],
    additionalColumns: []
}
return {
    tableActions: [
        <AssignmentsTableRefreshAction refresh={fetchTableData} />,
      ] ,
    additionalColumns: [{
          Header: '',
          Cell: AssignmentRowActionTableCell,
          id: 'action',
        }]
}
}
}